.github/workflows/ruby.yml

```
@@ -5,7 +5,7 @@
 # This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
 # For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
 
-name: Ruby
+name: Rails tests
```

If you rename this, you'd better rename the file as well.

I'd say

```suggestion
name: Ruby on Rails
```

and rename file to ruby-on-rails.yml